orchestra/migrations/0017_auto_20151007_1424.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
```

Comment:
this migration adds new models and changes existing models in a nullable/non-breaking way.